xl/settings.py

```diff
@@ -53,9 +62,11 @@
     'B': bool,
     'L': list,
     'D': dict,
-    'U': unicode
 }
 
+if PY2:
+    TYPE_MAPPING['U'] = unicode
```

Comment:
So... what happens if a user has a U setting, and migrates from py2 to py3? I suspect it will break?